internal/core/window.rs

```diff
@@ -811,6 +811,15 @@ impl WindowInner {
         if let Some(window_adapter) = window_adapter.internal(crate::InternalToken) {
             window_adapter.handle_focus_change(old, new);
         }
+
+        let popup_wa = self.active_popups.borrow().last().and_then(|p| match &p.location {
+            PopupWindowLocation::TopLevel(wa) => Some(wa.clone()),
+            PopupWindowLocation::ChildWindow(..) => None,
+        });
+        if let Some(popup_wa) = popup_wa {
+            // Also set the focus item on the inner window
+            popup_wa.window().0.set_focus_item(new_focus_item, set_focus);
```

Comment:
Do we still need to run all the other code in this function if we end up delegating to a popup's window adapter?

Reply:
I was also wondering. I can try.